src/Symfony/Bundle/FrameworkBundle/Tests/Command/RouterDebugCommandTest.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bundle\FrameworkBundle\Command\RouterDebugCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Routing\RouterInterface;
+
+class RouterDebugCommandTest extends TestCase
+{
+    public function testSortingByPriority()
+    {
+        $tester = $this->createCommandTester();
+        $result = $tester->execute(['--sort' => 'priority'], ['decorated' => false]);
+        $this->assertEquals(0, $result, 'Returns 0 in case of success');
```

Comment:
please use `assertSame()` everywhere if possible

Reply:
Ok! I used `RouterMatchCommandTest` as a template, do you think that a future refactoring PR for this test class might be accepted?